openstack/backup-replication/templates/service.yaml

```diff
@@ -4,6 +4,7 @@ metadata:
   name: backup-replication
   namespace: backup
   annotations:
+    checksum: {{ include "backup-replication/templates/replica-configmap.yaml" $ | sha256sum }}
```

Comment:
We don't need it on the service. Only for the deployment to have the spec changed.

Reply:
done